src/Zuora/Http/Request.php

```diff
@@ -31,6 +31,7 @@ function __construct($curl_options = array())
             'timeout' => 5,
             'ssl_verifypeer_skip' => false,
             'user_agent' => 'Zuora PHP Client/0.1',
+            'ssl_version' => 6, // TLS v1.2
```

Comment:
Can we use constant `CURL_SSLVERSION_TLSv1_2 ` here? http://php.net/manual/en/function.curl-setopt.php